pkg/scheduler/objects/application_state.go

```diff
@@ -143,49 +141,53 @@ func NewAppState() *fsm.FSM {
 				}
 			},
 			"leave_state": func(event *fsm.Event) {
-				event.Args[0].(*Application).clearStateTimer()
+				event.Args[0].(*Application).clearStateTimer() //nolint:errcheck
```

Comment:
Lets do this consistently:
* Remove all error checks on cast from all callback definitions including the string cast
* Add the nolint comment to all casts 
* Document at the start of the callback declaration the fact that arg 0 must be an app object and arg 1 a string. Doing something different will cause a run time panic.